src/common/components/profile-card/index.tsx

```diff
@@ -36,11 +36,13 @@ import {
     rssSvg,
 } from "../../img/svg";
 
+import { EditPic } from '../community-card'
+
 interface Props {
     global: Global;
     history: History;
     activeUser: ActiveUser | null;
-    account: Account;
+    account: any;
```

Comment:
Why type `any`?

Reply:
Neither Account nor FullAccount | BaseAccount were working for getting profile info. I work on it more.